clang/test/dpct/thrust-algo-raw-ptr-noneusm.cu

```diff
@@ -0,0 +1,438 @@
+// UNSUPPORTED: cuda-8.0
```

Comment:
can the migrated code can be build by icx?

Reply:
Yes，https://github.com/oneapi-src/SYCLomatic-test/pull/178 has been submitted to verify the build&run status.